src/main/java/io/dockstore/provision/S3CmdPlugin.java

```diff
@@ -67,12 +66,33 @@ public void stop() {
 
         private static final String CLIENT_LOCATION = "client";
         private static final String CONFIG_FILE_LOCATION = "config-file-location";
+        private static final String VERBOSITY = "verbosity";
         private static final String DEFAULT_CLIENT = "/usr/bin/s3cmd";
         private static final String DEFAULT_CONFIGURATION = System.getProperty("user.home") + "/.s3cfg";
+        private static final String DEFAULT_VERBOSITY = "2";
+        private int verbosity;
         private String client;
         private String configLocation;
         private Map<String, String> config;
 
+        public void setVerbosity(String verbosity) {
+            try {
+                switch (verbosity.toLowerCase()) {
+                case "minimal":
```

Comment:
Don't map between numbers and bare strings. 
Just use an enum, it will be easier for the next developer to understand the code.

Reply:
Current implementation similar to https://github.com/qos-ch/slf4j/blob/0b1e6d38cfabd4b7ed335aec1aa6b2ae0c770f08/slf4j-simple/src/main/java/org/slf4j/simple/SimpleLoggerConfiguration.java#L145

Reason I/we use numbers is so that when we have multiple level (like trace, debug, info, warn, and error) and a line is set to print on debug, we don't need to check by seeing if the current set enum is DEBUG or INFO or WARN or ERROR.  We can just do level > 2.  See: https://github.com/qos-ch/slf4j/blob/bc03976da8bc0b4e83bae3dc7404336e03e22017/slf4j-simple/src/main/java/org/slf4j/simple/SimpleLogger.java#L380

Reply:
Although I guess I can switch it to https://stackoverflow.com/questions/13070222/how-cant-we-compare-two-enum-values-with hmm...

Reply:
nvm, switched to enum